core-plugin/src/main/groovy/com/github/jrubygradle/api/gems/GemVersion.groovy

```diff
@@ -432,11 +432,21 @@ class GemVersion implements Comparable<GemVersion> {
     private static GemVersion parseTwiddleWakka(String singleRequirement) {
         String base = getVersionFromRequirement(singleRequirement, TWIDDLE_WAKKA)
         List<String> parts = base.tokenize(VERSION_SPLIT)
-        if (1 == parts) {
```

Comment:
That bug started me in the face and I did not even see it. :laughing:  It should originally have read `1==parts.size()`.